### ISSUE TITLE
Emscripten Examples Web Gallery - small issues list

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/openFrameworks#7827]
Examples Web Gallery is now working and up. 
https://openframeworks.cc/examples/ 

There are a few small issues in some examples and video playback isn't working though it should. 
Going to use this to track examples with issues:

Feel free to edit / add to this list: 

Broken:
- [ ] https://openframeworks.cc/examples/3d/ofxAssimpAdvancedExample/ - needs rebuilt assimp lib for glTF fix 
- [ ] same with: https://openframeworks.cc/examples/3d/assimp3DModelLoaderExample for second model 
- [ ] all examples that load video 
- [ ]

**Comment by artificiel** [comment]
the `communication` examples fail to start with various error messages (tried macOS safari and chrome) — but should they? is there a form of automatic websocket wrapper around sockets, and is UDP i/o expected to work without a form of proxy?

**Comment by ofTheo** [comment]
@artificiel yeah agree, I pulled the TCP ones, but I am pretty sure all the osc / udp ones don't make sense. I pulled the thread ones for similar reasons ( until we get a good thread/cors solution ) and windowing ones too.

**Comment by artificiel** [comment]
@ofTheo it would be "nice" to provide a proxy that someone can run to get UDP/OSC in and out of the TCP websockets (probably now requires wss:// which increases the complexity) of the browser, but it quickly raises the problem of managing networking expectations (where do you expect your OSC to be going from your local web browser?). if you are running an app in a browser to send OSC to 127.0.0.1 you might as well run a native app... (I think!?). conversely if you are running the proxy on a server (for a "public messaging type thing"), it will communicate with a single specific instance of the app at which point you also might as well run a native app? (unless the proxy copies to N dynamic websocket clients, which starts to look like an MQTT broker..). 

all this to say UDP over WSS is unlikely to fit realistic/non-naive scenarios.

**Comment by Jonathhhan** [comment]
@ofTheo the pointCloudExample issue could be solved with `glEnable(GL_POINT_SMOOTH);`
Now it is:
```
	#ifndef TARGET_EMSCRIPTEN
		glEnable(GL_POINT_SMOOTH); // use circular points instead of square points
		glPointSize(3); // make the points bigger
	#endif
```
Somehow the points are smaller than in https://openframeworks.cc/examples/3d/3DPrimitivesExample/ if I compile it, so `glEnable(GL_POINT_SMOOTH);` does not have an effect for me.

**Comment by ofTheo** [comment]
Yes. That’s correct. I couldn’t get GL_Points working at all so I did a separate implementation with triangle based billboards. 

So it will definitely look a bit different on web vs desktop. 

I thought about doing a geometry shader but probably we should emulate GL_Points internally via a shader or something.
